hebi_a-2084-01-parallel-gripper_moveit_config/config/ros_controllers.yaml

```diff
@@ -26,13 +26,17 @@ controller_list:
     default: True
     type: FollowJointTrajectory
     joints:
-      - HEBI/base/X5_4
-      - HEBI/shoulder/X5_9
-      - HEBI/elbow/X5_4
-      - HEBI/wrist1/X5_1
-  - name: hand_controller
-    action_ns: gripper_action
-    type: GripperCommand
-    default: true
-    joints:
-      - end_effector/input_l_finger
+      - Arm/J1_base/X5_4
+      - Arm/J2_shoulder/X5_9
+      - Arm/J3_elbow/X5_4
+      - Arm/J4_wrist1/X5_1
+hand_controller:
+  type: GripperCommand
+  joints:
+    end_effector/input_l_finger
+  gains:
+    end_effector/input_l_finger:
+      p: 100
+      d: 1
+      i: 1
+      i_clamp: 1
```

Comment:
The `hand_controller` changed quite a bit here -- I'm assuming this was intentional/following better practices now?  Just want to make sure it still works from MoveIt w/ the MoveIt arm node in sim and physical system